src/backend/distributed/shared_library_init.c

```diff
@@ -2058,6 +2058,7 @@ RegisterCitusConfigVariables(void)
 		GUC_STANDARD | GUC_NO_SHOW_ALL,
 		NULL, NULL, NULL);
 
+	/* deprecated setting */
```

Comment:
We seem to add `Deprecated` prefix to the internal variable as well: `DeprecatedDeferShardDeleteOnMove `

And, I guess we can make `ReplicateReferenceTablesOnActivate` static to prevent any confusion?